tracer/src/Datadog.Trace/RemoteConfigurationManagement/RcmCapabilitiesIndices.cs

```diff
@@ -45,6 +45,10 @@ internal static class RcmCapabilitiesIndices
 
         public static readonly BigInteger AsmCustomDataScanners = Create(17);
 
+        public static readonly BigInteger AsmExclusionData = Create(18);
```

Comment:
Is this one being introduced in a PR that's in development? I haven't seen this before

Reply:
There's a list of all capabilities here: https://datadoghq.atlassian.net/wiki/spaces/RC/pages/2594702792/Client+capabilities+list

I had to add this 18 when adding 19 because we have a unit test that doesn't allow leaving gaps. 🤷🏽‍♂️

Reply:
Or is this a standardized value across tracers and we just hadn't gotten to it yet?

Reply:
> is this a standardized value across tracers and we just hadn't gotten to it yet?

Correct.